src/solver/utils/ortools_utils.cpp

```diff
@@ -116,6 +116,20 @@ static void tuneSolverSpecificOptions(MPSolver* solver)
     }
 }
 
+static bool solverSupportsWarmStart(const MPSolver* solver)
+{
+    if (!solver)
+        return false;
+
+    switch (solver->ProblemType())
+    {
+    case MPSolver::XPRESS_LINEAR_PROGRAMMING:
+        return true;
+    default:
+        return false;
+    }
```

Comment:
So, there are no warm start for any solver apart from **xpress** behind **ortools**. I guess it's expected, but why ?

Reply:
Yes, this is expected. Methods `SetStartingLpBasisInt` and `GetFinalLpBasisInt` are implemented only for the LP XPRESS solver. For other solvers, a fatal log is emitted.